src/expr/impl/src/scalar/to_char.rs

```diff
@@ -44,7 +54,7 @@ impl ChronoPattern {
     /// Compile the pg pattern to chrono pattern.
     // TODO: Chrono can not fully support the pg format, so consider using other implementations
     // later.
-    pub fn compile(tmpl: &str) -> ChronoPattern {
+    pub fn compile(tmpl: &str, is_interval: bool) -> ChronoPattern {
```

Comment:
Perhaps keep the original `compile` and add a new `compile_for_interval`?

Reply:
done